CONTRIBUTING.md

```diff
@@ -1,13 +1,249 @@
+# Contributing to fqtk
 
-## Updating the README
+Thank you for your interest in contributing to fqtk! This document provides guidelines and information for contributors.
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [Getting Started](#getting-started)
+- [Development Workflow](#development-workflow)
+- [Issue Tracker](#issue-tracker)
+- [Pull Requests](#pull-requests)
+- [Coding Guidelines](#coding-guidelines)
+- [Testing](#testing)
+- [Documentation](#documentation)
+- [Release Process](#release-process)
+- [License](#license)
+
+## Code of Conduct
+
+We are committed to providing a welcoming and inclusive environment for all contributors. Please be respectful and considerate in all interactions.
```

Comment:
Aren't we already covered with our global code of conduct?

https://github.com/fulcrumgenomics/.github/blob/main/CODE_OF_CONDUCT.md